sbncode/CAFMaker/FillTrue.cxx

```diff
@@ -310,6 +310,7 @@ namespace caf {
       srneutrino.E = nu.Nu().EndMomentum().Energy();
       srneutrino.momentum = nu.Nu().EndMomentum().Vect();
       srneutrino.position = nu.Nu().Position().Vect();
+      srneutrino.time = nu.Nu().Position().T() / 1000. /* ns -> us */;
```

Comment:
I think this goes against the recent unit consistency efforts to keep things in units of GeV/cm/ns where reasonable. @PetrilloAtWork, @cjbackhouse , or @etyley could maybe reply.

Reply:
The timing is actually done in us rather than ns (This was more a documentation of what was currently in use rather than a deliberate decision): https://github.com/SBNSoftware/sbnanaobj/blob/feature/etyley_CAFUnits/sbnanaobj/StandardRecord/README.md

Reply:
Thank you! apologies for the noise.